File: sdks/python/v0/examples/sticky_workers/event.py
from dotenv import load_dotenv

from hatchet_sdk.hatchet import Hatchet

load_dotenv()

hatchet = Hatchet(debug=True)

# client.event.push("user:create", {"test": "test"})
hatchet.event.push(
    "sticky:parent",
    {"test": "test"},
    options={"additional_metadata": {"hello": "moon"}},
)

File: examples/typescript/durable-sleep/event.ts
import { hatchet } from '../hatchet-client';

async function main() {
  const event = await hatchet.events.push('user:event', {
    Data: { Hello: 'World' },
  });
}

if (require.main === module) {
  main()
    .then(() => process.exit(0))
    .catch((error) => {
      console.error('Error:', error);
      process.exit(1);
    });
}
